app/controllers/atlases_controller.rb

```diff
@@ -1,98 +1,143 @@
+    @atlas = Atlas.new(taxon_id: params[:taxon_id].to_i)
   end
-  
+
   def edit
-    @atlas = Atlas.find(params[:id])
     @exploded_atlas_places = @atlas.exploded_atlas_places.includes(:place)
     @atlas_places = @atlas.places
   end
-  
+
   def show
-    @atlas = Atlas.find(params[:id])
     @atlas_places = @atlas.places
-    @atlas_presence_places = @atlas.presence_places    
+    @atlas_presence_places = @atlas.presence_places
+
+    #any obs outside of the complete set
+    @observation_search_url_params = { 
+      taxon_id: @atlas.taxon_id, 
+      quality_grade: ["research","needs_id"].join(","), 
+      not_in_place: @atlas_presence_places.pluck(:id).join(",")
+    }
+    @num_obs = INatAPIService.observations(@observation_search_url_params.merge(per_page: 0)).total_results
     respond_to do |format|
       format.html do
-        @atlas_alterations = @atlas.atlas_alterations.includes(:place, :user).order("created_at DESC").limit(30).reverse
-        @atlas_place_json = {
-          type: "FeatureCollection", 
-          features: @atlas_places.select{|p| !(p.name == "Antarctica" && p.admin_level = 0) }.map{|p| 
-            {presence: (@atlas_presence_places.map(&:id).include? p.id) ? true : false, name: p.name, id: p.id, type: "Feature", geometry: RGeo::GeoJSON.encode(p.place_geometry.geom)}
-          }
-        }
+        @atlas_alterations = @atlas.atlas_alterations.includes(:place, :user).order("created_at DESC").
+          limit(30).reverse
+        @listed_taxon_alterations = @atlas.relevant_listed_taxon_alterations.includes(:place, :user).
+          order("listed_taxon_alterations.created_at DESC").limit(30).reverse
       end
-      format.json { render :json => @atlas_presence_places.to_json }
+      format.json { render json: @atlas_presence_places.to_json }
     end
   end
-  
+
   def create
     @atlas = Atlas.new(params[:atlas])
     respond_to do |format|
       if @atlas.save
-        format.html { redirect_to(@atlas, :notice => 'Atlas was successfully created.') }
+        format.html { redirect_to(@atlas, notice: 'Atlas was successfully created.') }
       else
-        format.html { render :action => "new" }
+        format.html { render action: "new" }
       end
     end
   end
-  
+
   def update
-    @atlas = Atlas.find(params[:id])
     respond_to do |format|
       if @atlas.update_attributes(params[:atlas])
         @atlas.taxon
-        format.html { redirect_to(@atlas.taxon || taxa_path, :notice => 'Atlas was successfully updated.') }
+        format.html { redirect_to(@atlas.taxon || taxa_path, notice: 'Atlas was successfully updated.') }
       else
-        format.html { render :action => "edit" }
+        format.html { render action: "edit" }
       end
     end
   end
-  
+
   def destroy
-    @atlas = Atlas.find(params[:id])
     @atlas.destroy
-    
     respond_to do |format|
       format.html { redirect_to(@atlas.taxon) }
     end
   end
-  
+
+  def map
```

Comment:
This is the action I made, but I think you've rolled it all over to atlases::show . If so, then this can go, as can the atlases/map.html.haml view, and the route